### DISCUSSION TITLE
리서치 소개

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/9th-builder#12] [Show and tell]
### 추천시스템 리서치
- 리서치 리더: 이경찬
- 리서치 빌더: 이경찬
- 리서치 소개: 지난 1년 반 동안 10편 이상의 추천시스템 주요 논문 리뷰 후 PseudoRec이라는 영화 추천시스템 사이트를 구현하며 개발 경험도 쌓았습니다. Sequential 및 Graph-based 등 최신 추천 모델과 CF, MF 등의 전통적인 추천 기법을 해당사이트에 구현하고 결과를 비교하며 토론합니다. 최근엔 LLM을 활용한 개인화 추천을 구현 중입니다. 또한 ‘월간슈도렉’이라는 이름으로 최신 추천시스템 관련 자료 리뷰, 또는 추천 모델 구현기를 월 1회 정리하며 Build in Public을 수행해 나갑니다.
- 진행중인 스터디/프로젝트
  - 추천시스템 논문에서 서비스까지 2(S8~ : 여름 계절학기 진행중)
- 완료한 스터디/프로젝트
  - 추천시스템 주요 논문 리뷰 및 구현(S6)
  - 추천시스템 논문에서 서비스까지 1(S7)
- 9기 모집 프로젝트
  - [Life is short, You need Recommendation system](https://github.com/Pseudo-Lab/9th-builder/discussions/21)
  - [내게 딱 맞는 건강디저트 추천시스템 구현하기](https://github.com/Pseudo-Lab/9th-builder/discussions/8)
- 주요 결과물 소개: https://www.pseudorec.com/
 
### 에이전트 빌더 길드
- 리서치 리더: 정한결
- 리서치 빌더: 정한결 (현재 논의중) 
- 리서치 소개: llm 엔지니어링, 자동화, 에이전트를 주제로 프로덕트화를 지향하는 리서치 팀입니다. 
참가자들은 llm 엔지니어링 관련된 논문을 읽어와 조사하고, 이를 자신의 프로덕트에 어떻게 구현했는지 결과를 발표합니다.  (즉, 매주 컨퍼런스식 발표를 하는 셈입니다).
궁극적으로 제작자들이 모여서 성장하는 도장,  또 장기적으로 협업할 수 있는 길드를 지향합니다. 
이번 시즌에는 llm agent 제작의 엔지니어링 부분에 대한 위키 핸드북을 낼 예정이고, 디스코드에서 하는 스터디를 외부 플랫폼에서 공개 송출해 볼 생각입니다.
- 진행중인 스터디/프로젝트명: 깃헙에 NLP 잔디심기 S5 
- 완료한 스터디/프로젝트명: 깃잔심S1(5기), 깃잔심S2(6기), 깃잔심S3(7기), 깃잔심S4(8기)
- 9기 모집 프로젝트
  - [깃헙에 NLP 잔디심기 5 :LLM Engineer Handbook 만들기! <경량화, 서빙, 로깅>](https://github.com/Pseudo-Lab/9th-builder/discussions/16)
  - [RAG is All You Need!](https://github.com/Pseudo-Lab/9th-builder/discussions/13)
  - [2024 초급 LLM with AI application](https://github.com/Pseudo-Lab/9th-builder/discussions/23)
- 주요 결과물 소개: (쥬피터 북 페이지 - 만드는 중!) 
- 협업/후원 기업: X, 추가 예정 
- 기타 추가사항: 슈도랩 외부의 그룹들과도 같이 작업하려고 합니다! 생기는 대로 공유할게요

### PseudoDiffusers
- 리서치 리더: 조상우
- 리서치 빌더: 조상우
- 리서치 소개: PseudoDiffusers 는 2년째 운영되고 있는 조직이며, 최근에는 CVPR 2024 Workshop 에 논문을 게재하기도 하였습니다. 스터디 주제로는 최신 text-to-image generation, text-to-video generation, 그리고 text-to-3d generation 모델들에 대해 논문 및 코드를 이론적으로 접근하며 리뷰합니다.
- 진행 중인 스터디: Text-to-Image Generation (feat.Diffusion) - Season 4 (진행 예정)
- 완료한 스터디/프로젝트명: Text-to-Image Generation (feat.Diffusion) - Season 1, Text-to-Image Generation (feat.Diffusion) - Season 2, Text-to-Image Generation (feat.Diffusion) - Season 3
- 9기 모집 프로젝트
  - [Text-to-Image Generation (feat. Diffusion) - Season 4](https://github.com/Pseudo-Lab/9th-builder/discussions/19)
- 주요 결과물 소개: DiffInject: Revisiting Debias via Synthetic Data Generation using Diffusion-based Style Injection (CVPR 2024 Workshop) 논문 게재, PseudoDiffusers 깃헙 블로그 운영 (https://github.com/Pseudo-Lab/pseudodiffusers)
- 협업/후원 기업: -
- 기타 추가사항: -
 
### Hugging Face KREW
- 리서치 리더: 정우준
- 리서치 빌더: 임지훈
- 리서치 소개: 🤗 Hugging Face 오픈소스에 기여하며 Hugging Face와 AI를 깊이 이해하고자 노력하는 리서치 팀입니다.
우리는 다음과 같은 활동을 통해 함께 성장하고 있습니다:
Hugging Face 기술 문서를 이해하고 번역하며 AI 지식을 확장합니다.
더 나아가, 직접 기술 문서를 작성하고 기여하며 Hugging Face 오픈소스에 가치를 더합니다.
모델 리소스 컨트리뷰션을 통해 다양한 딥러닝 모델들을 탐구하고, Hugging Face를 활용한 사용 방법을 연구합니다.
Hugging Face 코드를 분석하고 직접 코드 기여에 참여하여 개발 경험을 쌓고자 합니다.
이러한 활동을 통해 Hugging Face 한국 커뮤니티 활성화에 기여하고자 합니다. 우리의 경험과 지식을 공유하며 함께 배우고 발전하는 커뮤니티를 만들어 가는 것이 목표입니다.

- 진행 중인 스터디
  - OSSCA 2024에 Hugging Face OSS 팀으로 참가해 멘토링 진행 중.
- 완료한 스터디/프로젝트명: 
  - Hugging Face 온라인 문서 번역(S6)
  - OSSCA 2023 - Hugging Face 한글화 프로젝트(S7)
  - Hugging Face KREW Hackathon 2023: Everyday AI(🤗 Hugging Face 후원)
  - Hugging Face 공식 문서 번역(S8)
- 주요 결과물 소개: 
  - Transformers 문서 번역 (https://huggingface.co/docs/transformers/v4.43.4/ko/index)
  - Python Hub Library 문서 번역 (https://huggingface.co/docs/huggingface_hub/v0.24.5/ko/index)
  - Hugging Face와 협업해서 Hackathon 개최 (https://huggingface.co/collections/pseudolab/hugging-face-krew-hackathon-2023-652bb5066f88e29851af8076)
  - 번역 Glossary 사이트 (https://pseudo-lab.github.io/deep-learning-glossary/dictionary/glossary.html)
- 협업/후원 기업: 🤗 Hugging Facehttps://github.com/Pseudo-Lab/9th-builder/discussions/23

**Comment by jsshin2022** [comment]
### 인과추론 리서치 (Causal Inference Team)
- 리서치 리더 : [신진수](https://www.linkedin.com/in/jinsoo-shin-436060162/)
- 리서치 빌더 : [김성수](https://www.linkedin.com/in/%EC%84%B1%EC%88%98-%EA%B9%80-50825717b/), [박상호](https://www.linkedin.com/in/shstat1729/), [방태모](https://www.linkedin.com/in/taemo/), [이동욱](https://www.linkedin.com/in/dwlee-1995-lkdin/)
- 인과추론팀 Website
   - Github : https://github.com/CausalInferenceLab
   - 팀 페이지 : https://pseudo-lab.com/6bbf03d9f11d4af687c0f03c6db39b1b
- 인과추론팀 Vision : 한국어 자료가 부족한 인과추론 및 실험을 많은 분들이 쉽게 접하실 수 있도록 기여
- 인과추론팀 소개 : 인과추론팀은 22년 3월부터 시작되었으며, 가짜연구소에서 가장 많은 성과를 내고 오래 지속되고 있는 리서치팀입니다.  데이터를 통한 문제해결력을 높이기 위해 Causal Inference 학습 및 관련 프로젝트를 진행하고, 밋업 행사도 하며, 한국의 인과추론 분야를 리딩하고 있습니다. <실무로 통하는 인과추론 with 파이썬> 책을 출간하기도 하였으며, 번역 프로젝트가 깃헙에서 400+ 스타를 받았습니다. 24년에는 총 5명의 빌더와 40명 이상의 구성원이 인과추론 리서치팀으로 활동 중입니다.

- 인과추론팀에 참여한 구성원
   - 빌더 (5명) : 신진수, 김성수, 박상호, 방태모, 이동욱
   - 러너 (38명)
      - Since 2022 : 김상돈, 김성수, 김소희, 김준영, 남궁민상, 정호재, 최은희, 홍성철
      - Since 2023 : 김지연, 박병수, 박상우, 박시온, 박이삭, 유정현, 임종언, 이홍규, 조슬지, 조인서
      - Since 2024 : 김혜민, 문정하, 박수영, 박혜지, 손지우, 손지영, 심소현, 양유승, 안시형, 유도영, 유연승, 이명석, 이은경, 이재호, 최의관, 최지환, 하소희, 황영산, 황의림, 지서영, 조동민


- 진행중/진행 예정 프로젝트 
   - [Causal Inference + X](https://github.com/Pseudo-Lab/9th-builder/discussions/18) (9기, 김성수 빌더)
   - [Bridging Causal Inference and Real-World Evidence: A Study of FDA and Other Regulatory Guidelines](https://github.com/Pseudo-Lab/9th-builder/discussions/9) (9기, 박상호 빌더)
   - 실무로 통하는 인과추론 with 파이썬 특강 2nd (9기, 신진수 빌더)

- 완료한 프로젝트 (10개)
   - [Casual하게 Causality 이해하기](https://pseudo-lab.com/chanrankim/Casual-Causality-efb2d821ae334b8d9674ca331edebc72) (4기, 신진수 빌더)
   - [Casual하지 않게 Causality 이해하기](https://pseudo-lab.com/chanrankim/Casual-Causality-09ebf8f271784589bc18c38e1ea57b66) (5기, 신진수 빌더)
   - [Case로 Causality 이해하기](https://pseudo-lab.com/chanrankim/Case-Causality-5030892fed8d4b14842e526d4b40c2b9) (6기, 신진수 빌더)
   - [인과추론 라이브러리 한국어 자료 만들기](https://pseudo-lab.com/chanrankim/Social-Science-bb2c83819f744e1eaa0ddc76961a7dd5) (7기, 신진수 빌더)
   - [Gamifying Causal Inference with Game](https://pseudo-lab.com/chanrankim/Entertainment-Gamifying-Causal-Inference-with-Game-105f4d2a7ee64c2ea58f3757cbf12b32) (7기, 신진수 빌더)
   - [온라인 통제 실험 연구자로 거듭나기](https://github.com/CausalInferenceLab/OCE-Materials) (8기, 방태모 빌더)
   - [실무로 통하는 인과추론 with 파이썬 특강 1st](https://github.com/CausalInferenceLab/causal-inference-lecture) (8기, 신진수 빌더)
   - [Smart Causal Inference](https://github.com/CausalInferenceLab/Smart_Causal_Inference) (8기, 박상호 빌더)
   - [인과추론과 실무](https://github.com/CausalInferenceLab/causal-inference-practice) (8기, 김성수 빌더)
   - [Kafka - Zero to Hero 실시간 성장하기](https://github.com/Pseudo-Lab/data-engineering-for-everybody) (8기, 이동욱 빌더)

- 주요 결과물 
   - [Causal Inference for the brave and true 오픈 소스 번역](https://github.com/CausalInferenceLab/Causal-Inference-with-Python) (Github Star 400+)
   - [실무로 통하는 인과추론 with 파이썬 출간 (한빛미디어, 2024.03)](https://product.kyobobook.co.kr/detail/S000212577153)
   - [실무로 통하는 인과추론 with 파이썬 무료 Youtube 특강 제작](https://www.youtube.com/playlist?list=PLKKkeayRo4PWuwIL0_C7n_QQQ_at6CpoE) (with 박지용 교수님)
   - [Korea Summer Workshop on Causal Inference 2024](https://sites.google.com/view/causal-inference2023) 협력 / 기획 (신진수) / 연사 발표 (방태모) 
   - [인과추론팀 자체 행사 개최 : "인과추론으로 네트워킹하기"](https://pseudo-lab.com/chanrankim/2024-9ee75a9ff9004d23945fe8fd421d37aa) (신청자 82명 중, 75명 참석) 
 
- 후원 기업 : [한빛미디어](https://www.hanbit.co.kr/)
